Sources/Danger/GitHubDSL.swift

```diff
@@ -91,17 +92,17 @@ public struct GitHubPR: Decodable {
     /// The users who are assigned to the pull request.
     public let assignees: [GitHubUser]
 
-    /// The ISO6801 date string for when the pull request was created.
-    public let createdAt: String
+    /// The ISO8601 date string for when the pull request was created.
+    public let createdAt: Date
 
-    /// The ISO6801 date string for when the pull request was updated.
-    public let updatedAt: String
+    /// The ISO8601 date string for when the pull request was updated.
```

Comment:
The documentation for all these still reference `String` after you've changed the type to `Date`

Reply:
@d-date `The ISO8601 date string for when` -> `The date when`
Very nitty, I know